### ISSUE TITLE
Smartphone validation: Switch to new message format

### ISSUE THREAD

**Issue by yamgent** [yamgent/CS3235_Project_Server#4]
New message format is as follows:
```json
{
    "secret_key": "ASDFHAKJDHGKJHDWSKFHSD",
    "success": true
}
```

**Comment by yamgent** [comment]
Documentation can be found here:
https://github.com/CS3235-1718-SEM1/CS3235_Data_Access_Layer/wiki/RESTful-API

**Comment by yamgent** [comment]
Fixed with 2581df29c6f95f47f51bb2c925eacd6c18ba842d.
